src/pages/loginView.js

```diff
@@ -58,7 +58,21 @@ export default function LoginView() {
   };
 
   /**
-   * Reset password
+   * handle google authentication
+   */
+  const googleLoginHandler = async () => {
```

Comment:
This code should be moved into `authService.js`, where all existing Firebase authentication stuff lives.

Reply:
Thanks for your review, I will take a look at how sign up is worked and adjust my function.